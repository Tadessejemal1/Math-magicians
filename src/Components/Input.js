const Input = () => (
  <div className="input">0</div>
);

export default Input;
