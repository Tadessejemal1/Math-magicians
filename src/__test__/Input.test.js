import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../Components/Input';

it('Input component renders correctly', () => {
  const tree = renderer
    .create(<Input text="8" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
