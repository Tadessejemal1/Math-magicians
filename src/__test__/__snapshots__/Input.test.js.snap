// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component renders correctly 1`] = `
<div
  className="input"
>
  8
</div>
`;

exports[`Input component renders correctly 1`] = `
<div
  className="input"
>
  8
</div>
`;
